README.md

```diff
@@ -1,17 +1,93 @@
 [![Build Status](https://travis-ci.org/Nexapp/NexappCore.svg?branch=master)](https://travis-ci.org/Nexapp/NexappCore)
 
 # Nexapp Core
-Several classes that we use everyday.
+Several Java mini-modules for REST APIs.
```

Comment:
Technically there's still nothing here for REST APIs

Reply:
Not about Jersey, but most classes are used in our APIs.